packages/scaffolding-cli/templates/src/ssr/components/Header/index.tsx

```diff
@@ -18,7 +18,7 @@ export const Header = () => {
     return (
         <AppBar position="fixed" color="secondary">
             <Toolbar>
-                <Link href="/blog">
+                <Link href="/cms">
```

Comment:
oh yeah I think this can stay as blog if our sample implementation is a blog

Reply:
> I think we should move the CMS folder out of the pages and into a separete folder within the ssr and decribe that by placing it inside the pages you must have a Contenful integration set up + required env keys for it to compile

So by default we turn off the CMS then?  Only turn it on demand?